components/DynamicForm/MyContinue.tsx

```diff
@@ -13,7 +13,7 @@ Represents a "Continue" question in the form, which just displays some informati
 const MyContinue: React.FC<ContinueProps> = ({ label, onMount }) => {
   useEffect(() => {
     onMount()
-  }, [onMount])
```

Comment:
Just curious, what purpose did OnMount serve?

Reply:
`onMount` is a custom function we had passed in from our [ChooseFormType](https://github.com/sandboxnu/edulaw/blob/56392eb02a73753775464fb46344400bd8ba7a49/components/DynamicForm/ChooseFormType.tsx#L61-L73) which actually referenced a function passed in from our [DynamicPOC](https://github.com/sandboxnu/edulaw/blob/56392eb02a73753775464fb46344400bd8ba7a49/pages/form/index.tsx#L217-L222), corresponding to our `setCurrentAnswer`

In the other form question types this helps us update the current answer to whatever the user has selected while maintaining state in DynamicPOC. However, in this component there is no current answer to select, so when the component loads we want to select a pseudoanswer on behalf of the user, which will allow us to do two things: 1) store the "answer" in our form values/history and 2) redirect the user to the correct question after pressing continue.

The reason `onMount` was removed from the dependency list is actually just something wack with React where calling a function causes it to change in the eyes of the `useEffect`
Before: `onMount` in dependency list, every time the `useEffect` runs it will trigger `onMount`, which will cause `useEffect` to run again, etc. etc. infinite looping
After: `onMount` removed from dependency list, `useEffect` runs when component is rendered on the page once and not again